server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/MessageAppender.java

```diff
@@ -58,7 +59,7 @@ public MessageAppender(AttachmentManager attachmentManager, MIMEMessageConverter
         this.mimeMessageConverter = mimeMessageConverter;
     }
 
-    public MessageFactory.MetaDataWithContent appendMessageInMailbox(ValueWithId.MessageWithId.CreationMessageEntry createdEntry,
+    public MessageFactory.MetaDataWithContent appendMessageInMailbox(CreationMessageEntry createdEntry,
```

Comment:
the remark was about importing ValueWithId, not MessageWithId, so you keep MessageWithId.CreationMessageEntry

Reply:
Sorry I don't get it. Naming **CreationMessageEntry** seems to convey enough meaning, I believe we can get a ride of **MessageWithId**